index.js

```diff
@@ -1 +1,95 @@
-console.log('hello')
\ No newline at end of file
+let gameFlow = document.getElementById('game')
+let scoreElement = document.getElementById('score')
+let scoreCounter = 0 // for score
+
+const gameFlowHeight = parseInt(window.getComputedStyle(gameFlow).getPropertyValue('height'))
+
+const blockClasses = ['left-arrow', 'up-arrow', 'down-arrow', 'right-arrow']
+
+class Block {
+    id = Date.now
+    className = getRandom().itemClass
+
+    createItem() {
+        const newItem = document.createElement('div')
+        newItem.setAttribute('class', `${this.className} drop-block`)
+        newItem.setAttribute('id', `${this.id}`)
+        return newItem
+    }
+}
+
+
+const checkTouch = (item) => {
+    const itemPosition = parseInt(window.getComputedStyle(item).getPropertyValue('top'))
+    const grMinHeight = 90
+    const grMaxHeight = 60
+    const goodMinHeight = 110
+    const goodMaxHeight = 30
+    const greatArea = itemPosition >= gameFlowHeight - grMinHeight && itemPosition <= gameFlowHeight - grMaxHeight
+    const goodArea = itemPosition >= gameFlowHeight - goodMinHeight && itemPosition <= gameFlowHeight - goodMaxHeight
+
+    if (greatArea) {
+        scoreCounter += 2
+    } else if (goodArea) {
+        scoreCounter += 1
+    }
+    scoreElement.textContent = scoreCounter
+}
+
+const getRandom = () => {
+    const elements = Math.floor((Math.random() * 20) + 5)
```

Comment:
What are 20 and 5?

Reply:
@silchencko its for number of min and max elements (which are created and fall). 5 - min, 20 (+5) - max
I thought that these magic numbers can still be left in order not to create special variables for the minimum/maximum for elements, timer and class names

Reply:
`elements` is a number? It still takes some time for me to understand what this expression does. Is it a random amount of arrows?